robottelo/ui/hosts.py

```diff
@@ -291,3 +291,9 @@ def fetch_host_parameters(self, name, domain_name, parameters_list):
             self.click(tab_locator)
             result[param_name] = self.wait_until_element(param_locator).text
         return result
+
+    def get_host_property(self, parameter_name):
```

Comment:
I am sorry man, but it is completely different things =) ones that you see through 'edit' and 'view/open' interfaces